src/components/PersonLink.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Person } from '../types';
+import { Link, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const [searchParams] = useSearchParams();
+  const { name, sex, slug } = person;
+
+  return (
+    <Link
+      className={classNames({
+        'has-text-danger': sex === 'f',
+        '': sex === 'm',
```

Comment:
The classNames function includes an unnecessary empty string for the male gender ('': sex === 'm'). This can be omitted as it doesn't add any class and could lead to confusion.